parts/dcoscustomdata173.t

```diff
@@ -1,342 +1,348 @@
-- content: '[Service]
-
-    Restart=always
-
-    StartLimitInterval=0
-
-    RestartSec=15
-
-    ExecStart=
-
-    ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=overlay
-
-    '
-  path: /etc/systemd/system/docker.service.d/execstart.conf
-  permissions: '0644'
-- content: '[Unit]
-
-    PartOf=docker.service
-
-    [Socket]
-
-    ListenStream=/var/run/docker.sock
-
-    SocketMode=0660
-
-    SocketUser=root
-
-    SocketGroup=docker
-
-    ListenStream=2375
-
-    BindIPv6Only=both
-
-    [Install]
-
-    WantedBy=sockets.target
-
-    '
-  path: /etc/systemd/system/docker.socket
-  permissions: '0644'
-- content: '[Unit]
-
-    Requires=dcos-setup.service
-
-    After=dcos-setup.service
-
-    [Service]
-
-    Type=oneshot
-
-    EnvironmentFile=/etc/environment
-
-    EnvironmentFile=/opt/mesosphere/environment
-
-    ExecStart=/usr/bin/bash -c "echo -e \"127.0.0.1 localhost\n$(detect_ip) $(hostname)\"
-    > /etc/hosts"
-
-    '
-  path: /etc/systemd/system/dcos-config-writer.service
-  permissions: '0644'
-- content: 'MESOS_CLUSTER={{{masterPublicIPAddressName}}}
-
-    '
-  path: /etc/mesosphere/setup-packages/dcos-provider-azure--setup/etc/mesos-master-provider
-- content: '
-
-    ADMINROUTER_ACTIVATE_AUTH_MODULE={{{oauthEnabled}}}
-
-    '
-  path: /etc/mesosphere/setup-packages/dcos-provider-azure--setup/etc/adminrouter.env
-- content: '["'', DCOSCUSTOMDATAPUBLICIPSTR''"]
-
-    '
-  path: /etc/mesosphere/setup-packages/dcos-provider-azure--setup/etc/master_list
-- content: 'EXHIBITOR_BACKEND=AZURE
-
-    AZURE_CONTAINER=dcos-exhibitor
-
-    AZURE_PREFIX={{{masterPublicIPAddressName}}}
-
-    '
-  path: /etc/mesosphere/setup-packages/dcos-provider-azure--setup/etc/exhibitor
-- content: 'com.netflix.exhibitor.azure.account-name={{{masterStorageAccountName}}}
-
-    com.netflix.exhibitor.azure.account-key='', listKeys(resourceId(''Microsoft.Storage/storageAccounts'',
-    variables(''masterStorageAccountName'')), ''2015-06-15'').key1,''
-
-    '
-  path: /etc/mesosphere/setup-packages/dcos-provider-azure--setup/etc/exhibitor.properties
-- content: '{"uiConfiguration":{"plugins":{"banner":{"enabled":false,"backgroundColor":"#1E232F","foregroundColor":"#FFFFFF","headerTitle":null,"headerContent":null,"footerContent":null,"imagePath":null,"dismissible":null},"branding":{"enabled":false},"external-links":
-    {"enabled": false},
-
-
-    "authentication":{"enabled":false},
-
-
-    "oauth":{"enabled":{{{oauthEnabled}}},"authHost":"https://dcos.auth0.com"},
-
-
-
-    "networking":{"enabled":false},"organization":{"enabled":false},"tracking":{"enabled":false}}}}
-
-    '
-  path: /etc/mesosphere/setup-packages/dcos-provider-azure--setup/etc/ui-config.json
-- content: '{}'
-  path: /etc/mesosphere/setup-packages/dcos-provider-azure--setup/pkginfo.json
-- content: ''
-  path: /etc/mesosphere/roles/azure
-- content: 'PROVISION_STR'
-  path: "/opt/azure/containers/provision.sh"
-  permissions: "0744"
-  owner: "root"
+bootcmd:
```

Comment:
In the diff for this file, I see this:

```
+- content: 'DCOS_ENVIRONMENT={{{targetEnvironment}}}
 +
 +    '
 +  owner: root
 +  path: /etc/mesosphere/setup-flags/dcos-deploy-environment
 +  permissions: '0644'
```

Which doesn't seem like quite what you wanted? Otherwise, makes sense. thanks.